stdlib/http/src/main/ballerina/http/auth/authn_handler_chain.bal

```diff
@@ -46,7 +46,10 @@ function AuthnHandlerChain::handle (Request req) returns (boolean) {
         var authnHandler = <HttpAuthnHandler> currentAuthHandler;
         if (authnHandler.canHandle(req)) {
             log:printDebug("Trying to authenticate with the auth provider: " + currentAuthProviderType);
-            return authnHandler.handle(req);
+            boolean authnSuccessful = authnHandler.handle(req);
```

Comment:
Isn't better to add a comment here explaining the rationale behind this logic?

Reply:
+1